### PR TITLE
INSP: Add option to enable unused import inspection in doctests

### PR DESCRIPTION
**Pull request by dima74**
changelog: Add option to enable unused import inspection in doctests

### CODE REVIEW

File: src/main/kotlin/org/rust/ide/inspections/lints/RsUnusedImportInspection.kt [hunk 1]
```diff
@@ -89,6 +94,10 @@ class RsUnusedImportInspection : RsLintInspection() {
         )
     }
 
+    override fun createOptionsPanel(): JComponent = MultipleCheckboxOptionsPanel(this).apply {
+        addCheckbox("Ignore doctests", "ignoreDoctest")
```
Comment (Kobzol) [inline src/main/kotlin/org/rust/ide/inspections/lints/RsUnusedImportInspection.kt:98]: This name seems slightly confusing to me. Maybe something more along the line of `Disable inside doctests` or `Ignore unused imports in doctests`?

**Comment by Kobzol** [comment]
The quick fix to remove an unused import does not work inside doctests :(

**Review by Kobzol — APPROVED** [review]
bors r+

**Comment by dima74** [comment]
Quick inside doctest indeed doesn't work, and looks like that any quick fix which removes lines doesn't not work inside doctests (#7565). So for now I disabled the quick-fix inside doctests (but keep the inspection)

**Pull request merged**